source/_components/bmw_connected_drive.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "BMW connected drive"
+description: "Instructions on how to setup your BMW connected drive account with Home Assistant."
+date: 2018-01-10 23:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.65
+---
+
+This component lets you retrieve data on your BMW vehicle from the BMW Connected Drive portal. You need to have a working BMW Connected Drive account and a Connected Drive enabled vehicle for this to work.
+
+For compatibility with your BMW vehicle check the (bimmer_connected page)[https://github.com/ChristianKuehnel/bimmer_connected] on github.
+
+To enable this component in your installation, add the following to your
+`configuration.yaml` file:
+
+{% raw %}
```

Comment:
I think you only need raw tags around templates. Someone can correct me if I'm wrong.

Reply:
ok, removed them